my_examples/ex1.py

```diff
@@ -0,0 +1,31 @@
+import loop_tool_py as lt
```

Comment:
I think we can skip checking in this folder

Reply:
Yeah, I mistakenly added this example.